mkdocs.yml

```diff
@@ -34,6 +34,14 @@ theme:
       toggle:
         icon: material/brightness-4
         name: Switch to light mode
+copyright: Copyright &copy 2023 dynamiqs
+extra:
+  generator: false
```

Comment:
Mmmm, have you read the "Please read this before removing the generator notice" part here > https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/?h=#generator-notice? I think we should either sponsor mkdocs-material, either keep the notice.